### PR TITLE
Create ChatWork.gs

### PR DESCRIPTION
**Pull request by kyo-ago**

### CODE REVIEW

File: samples/ChatWork.gs [hunk 1]
```diff
@@ -0,0 +1,83 @@
+var CLIENT_ID = '...';
+var CLIENT_SECRET = '...';
+
+/**
+ * Authorizes and makes a request to the Harvest API.
```
Comment (erickoledadevrel) [inline samples/ChatWork.gs]: Please fix the API name in this comment.

File: samples/ChatWork.gs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+ */
+function run() {
+    var service = getService();
```
Comment (erickoledadevrel) [inline samples/ChatWork.gs]: This library uses 2-space tabs throughout. Please update this sample to match.

File: samples/ChatWork.gs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        // Set the name of the callback function that should be invoked to
+        // complete the OAuth flow.
+        .setCallbackFunction('authCallback')
+
+        .setScope(scope)
+
+        .setTokenHeaders({
+            'Authorization': 'Basic ' + Utilities.base64Encode(CLIENT_ID + ':' + CLIENT_SECRET)
+        })
+        .setTokenPayloadHandler(function(tokenPayload) {
```
Comment (erickoledadevrel) [inline samples/ChatWork.gs]: Could you add a comment that explains why this is needed?

**Comment by kyo-ago** [comment]
Thanks for review!
And I fixed.

**Comment by erickoledadevrel** [comment]
Looks great, thanks!

**Pull request merged**